### ISSUE TITLE
Create Nuget Packages

### ISSUE THREAD

**Issue by yawkale** [yawkale/nop-umbraco-integration#15]

**Comment by pkmccaffrey** [comment]
Will the nuget packages allow creating a "new" project (not using the existing sample databases)?

**Comment by yawkale** [comment]
Yes,  I plan 2: first for new projects and second with sample data

**Comment by pkmccaffrey** [comment]
Excellent.  Let me know if there's anything I can do to help!
